earth_enterprise/src/server/wsgi/serve/publish/publish_app.wsgi

```diff
@@ -1 +1,4 @@
+import os
+os.environ['PYTHONHTTPSVERIFY'] = '0'
```

Comment:
may be add a comment explaining this change... ex: this is a temp fix for #issueNumber to disable ...